app/master/cluster_master.py

```diff
@@ -25,26 +22,13 @@ class ClusterMaster(object):
 
     def __init__(self):
         self._logger = get_logger(__name__)
-
-        self._all_slaves_by_url = {}
-        self._all_builds_by_id = OrderedDict()  # This is an OrderedDict so we can more easily implement get_queue()
-        self._builds_waiting_for_slaves = Queue()
-
-        self._request_queue = Queue()
-        self._request_handler = SerialRequestHandler()
-
-        self._request_queue_worker_thread = SafeThread(
-            target=self._build_preparation_loop, name='RequestHandlerLoop', daemon=True)
-        self._request_queue_worker_thread.start()
-
-        self._slave_allocation_worker_thread = SafeThread(
-            target=self._slave_allocation_loop, name='SlaveAllocationLoop', daemon=True)
-        self._slave_allocation_worker_thread.start()
-
         self._master_results_path = Configuration['results_directory']
-
+        self._all_slaves_by_url = {}
+        # This is an OrderedDict so we can more easily implement get_queue()
```

Comment:
I think you can remove this comment -- we don't even have a `get_queue()` anymore. :cookie: